float32 sensor_data 
---
geometry_msgs/PoseStamped goal
